<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wp3_4</class>
 <widget class="QWizardPage" name="Wp3_4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox_35">
     <property name="text">
      <string>Требуются дополнительные исследования для выявления симптомов болезней пульпы и периодонтальных тканей зубов (К04.) </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radioButton_8">
     <property name="text">
      <string>Нет болезней твердых тканей зубов </string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox_31">
     <property name="text">
      <string>Боль</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Болезни пульпы и периодонтальных тканей зубов (КО4)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBox_32">
     <property name="text">
      <string>Отек</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Симптомы болезней твердых тканей зубов</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkBox_34">
     <property name="text">
      <string>Разрушение коронки зуба</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBox_33">
     <property name="text">
      <string>Большая пломба</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
